File: 800/1743A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int t;
    cin >> t;

    while (t--) 
    {
        int n;
        cin >> n;

        int cnt = 10 - n;

        // accepting values is just formality
        for (int i = 1; i <= n; i++) 
        {
            int temp;
            cin >> temp;
        }

        cout << (6 * cnt * (cnt - 1)) / 2 << "\n";
    }

    return 0;
}

File: 800/1743A/main.java
import java.util.*;
public class main
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        
        while(t-- > 0)
        {
            int n = sc.nextInt();
            
            int cnt = 10-n;
            
            // accepting values is just formality
            for(int i=1;i<=n;i++)
                int temp = sc.nextInt();
                
            System.out.println((6*cnt*(cnt-1))/2);
        }
    }
}